services/dask-sidecar/Dockerfile

```diff
@@ -125,9 +125,9 @@ COPY --chown=scu:scu services/dask-sidecar/docker services/dask-sidecar/docker
 # make sure dask worker is started as ``dask-worker --dashboard-address 8787``.
 # Otherwise the worker will take random ports to serve the /health entrypoint.
 HEALTHCHECK \
-  --interval=60s \
-  --timeout=60s \
-  --start-period=10s \
+  --interval=30s \
```

Comment:
what is the criteria you used to decide these values?

Reply:
well I could return the question... they were set higher than most of the others we have...

Reply:
so I use more or less the time to start in production.